README.md

```diff
@@ -1003,6 +1003,43 @@ const MyResponsiveDiv = glamorous.div({
 ```
 </details>
 
+In addition to all this, you can reference a `glamorous` component inside the
+styles of other `glamorous` components (this works because they each implement
+the `toString` method):
+
+<details>
+<summary>Contextual Overrides</summary>
+
+This API is inspired by `styled-components`.
+
+```javascript
+const Link = glamorous.a({
+  color: 'inherit',
+  textDecoration: 'none',
+  '&:hover, &:active': {
+    textDecoration: 'underline',
+  }
+})
+
+const P = glamorous.p({
+  [`& ${Link}`]: {
```

Comment:
Unfortunately we cannot support:

```js
const P = glamorous.p({
  [Link]: {}
})
```

Because that would be like:

```javascript
const P = glamorous.p({
  '.g-1-1': {},
})
```

Which is invalid glamor syntax :-(